django/db/migrations/autodetector.py

```diff
@@ -1460,6 +1460,29 @@ def _constraint_should_be_dropped_and_recreated(
         for attr in new_constraint.non_db_attrs:
             new_kwargs.pop(attr, None)
 
+        # Replace renamed fields if the db_column is preserved.
+        for (
+            rem_app_label,
+            rem_model_name,
+            rem_db_column,
+            rem_field_name,
+            app_label,
+            model_name,
+            field,
+            field_name,
+        ) in self.renamed_operations:
```

Comment:
I know this isn't your doing but I really dislike excessively large (& overused for convenience sake) tuples.  This is why Raymond Hettinger created named tuples 😅
```suggestion
        for (
            _,
            _,
            rem_db_column,
            rem_field_name,
            _,
            _,
            field,
            field_name,
        ) in self.renamed_operations:
```

This patch is a little hard to grok at first glances, my recommendation is to replace the unused vars being unpack with `_` 🤷  Some folks hate that but I tend to think it communicates "We're not using this so don't worry about it"